src/sources/socket/mod.rs

```diff
@@ -320,6 +320,7 @@ mod test {
     }
 
     #[tokio::test]
+    #[ignore] // see issue #9046
```

Comment:
Could we drop this ignore now that it is fixed on master?

Reply:
Ah yeah, thank you.

Reply:
Resolved in 8552aab07